apps/meteor/client/NavBarV2/NavBarVoiceCallToolbar/index.ts

```diff
@@ -0,0 +1 @@
+export { default as NavBarItemVoiceCallDialer } from './NavBarItemVoiceCallDialer';
```

Comment:
Since this is an index file for NavBarVoiceCallToolbar folder, in the future new toolbar items could be added. Changing it to return a default doesn't seem appropriate.  This follows the standard already being used by the other toolbars in `NavBar.tsx`. 
Ex.:
```javascript
import { NavBarItemVoiceCallDialer } from './NavBarVoiceCallToolbar'
```